### DISCUSSION TITLE
Urgent Assistance Required for GitHub Account help please

### DISCUSSION THREAD

**Discussion by haloxtradee** [community/community#135865] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

account @budalebah flagged , this account has multiple contributions in projects and now account down. this situatin effects many projects. please help to un flag it. after a project started to a workflow accoount flagged. this is an urgent call.
this account  greatly value the services provided by GitHub and contrituting to many projects.  If there are any additional details or documentation required to support  please let me know, and I will be happy to provide them.

Thank you for your attention to this matter. I look forward to your prompt response.

**Comment by LiteBrite82** [comment]
Hi @haloxtradee ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by haloxtradee** [reply]
I have pro but cant get support via mail, how to fix this

**Reply by haloxtradee** [reply]
Really unbeliveble, i will lost my job

**Reply by haloxtradee** [reply]
@LiteBrite82 please help me

**Comment by haloxtradee** [comment]
@mgriffin @zackfern i need urgent help

**Reply by haloxtradee** [reply]
@turbo @keisaacson @timoyates @samus-aran 

I need urgent help for my account @budalebah please help me to reopen it, or i will lost my job

**Comment by eugenekartashian** [comment]
create new account and ask support to merge it, but it will take som time

**Reply by haloxtradee** [reply]
This is my new account my main account is @budalebah ser

**Reply by haloxtradee** [reply]
And i have been waiting almost 3 weeks

**Comment by Giddy-K** [comment]
If your GitHub account has been flagged, the best course of action is to contact GitHub Support directly. They are the only ones who can resolve issues related to account suspensions or flags, you can emphasize the urgency due to the impact on ongoing projects.

https://support.github.com/request

**Reply by haloxtradee** [reply]
Already done, no reply

**Comment by haloxtradee** [comment]
@still-hash-ir how many days did it take to reopen ?

**Comment by LiteBrite82** [comment]
Hi @haloxtradee 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
